src/TTGamesExplorerRebirthLib/Formats/NuCore/NuTexGenHdr.cs

```diff
@@ -38,16 +38,16 @@ public NuTexGenHdr Deserialize(BinaryReader reader, uint nuTexHdrVersion)
                 {
                     path = reader.ReadSized32NullTerminatedString();
 
-                    uint nuTexGenHdrUnknown1 = reader.ReadUInt32BigEndian();
+                    uint nuttype = reader.ReadUInt32BigEndian();
                 }
 
                 if (nuTexHdrVersion == 12)
                 {
-                    uint nuAlignedBuffer = reader.ReadUInt32();
+                    uint Level = reader.ReadUInt32(); // Accurev internal level (unused)
```

Comment:
```suggestion
                    uint level = reader.ReadUInt32(); // Accurev internal level (unused)
```
What’s « Accurev » ?

Reply:
their internal source control system